android/build.gradle

```diff
@@ -30,6 +30,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.19.+'
+    compile 'com.facebook.react:react-native:0.32.+'
```

Comment:
Hi again. It turns out that I forgot to link the library after upgrading to RN 0.33. Now it works fine. I guess that version restriction in the gradle file isn't really important as RN fetches the dependencies via NPM.